### DISCUSSION TITLE
Is it possible to embed it in a QWebEngineView

### DISCUSSION THREAD

**Discussion by tamlok** [Milkdown/milkdown#93] [Q&A]
Hi,

I want to introduce a powerful WYSIWYG Markdown editor into my project [VNote](https://github.com/vnotex/vnote). VNote has a Chrome kernel for its read mode. Will it be possible to embed it into VNote? I think if it is simple and browser-friendly, we could do this.

Thanks!

**Comment by Saul-Mirone** [comment]
Absolutely you can.
What you need is to follow these steps:

Create an empty directory.
1. Using [npm](https://www.npmjs.com/) to install milkdown core, preset, plugin and themes you need. (You can use npm if you have nodejs installed).
2. You can find the .js and .css style file in: `node_modules/@milkdown/preset-gfm(for example)/lib/`
3. If you prefer to combine them into one file, you need a front-end package tool, like webpack or vitejs. This may need some front-end tool chain knowleadge.

**Reply by tamlok** [reply]
Thanks! Will report here once succeeded.

**Comment by Saul-Mirone** [comment]
@tamlok Moved to discussion.
